cirq/devices/noise_model.py

```diff
@@ -133,6 +136,8 @@ def _noisy_moment_impl_moments(self, moment: 'cirq.Moment',
     def _noisy_moment_impl_operation(self, moment: 'cirq.Moment',
                                      system_qubits: Sequence['cirq.Qid']
                                     ) -> 'cirq.OP_TREE':
+        if self.is_virtual_moment(moment):
```

Comment:
Worth a short comment to explain why we ignore noise in virtual moments?

Reply:
Done.